rust/src/measures/ffi.rs

```diff
@@ -74,80 +74,92 @@ pub extern "C" fn opendp_measures__measure_distance_type(
     )))
 }
 
-#[bootstrap(returns(c_type = "FfiResult<AnyMeasure *>"))]
+#[bootstrap(
+    generics(T(default = "f64")),
+    returns(c_type = "FfiResult<AnyMeasure *>")
+)]
 /// Construct an instance of the `MaxDivergence` measure.
 ///
 /// # Arguments
-/// * `T` - The type of the distance.
-fn max_divergence<T>() -> MaxDivergence<T> {
-    MaxDivergence::default()
+/// * `T` - DEPRECATED. The type of the distance.
```

Comment:
What if you just got rid of the type parameter? Or if that's not good, file a follow-up issue to remove this at some point?